communications/slack-culture.rst

```diff
@@ -83,6 +83,31 @@ You can save space, time, and distractions to others by using commonly understoo
 
 -  ``:thank_you:`` (🙏) or ``:thankyou:``
 
+.. _slack-direct-messages:
+
+Direct messages
+---------------
+
+Please keep direct messaging to a user or a set of users to minimum.
+Direct messages should be avoided for any technical discussions or asking for support, even if you are sure that you have involved all those who are interested or who can contribute.
+Instead, such discussions should be in a thread in an appropriate :ref:`discussion channel<slack-discussion-channels>`.
+Direct messages shoul be reserved for conversations on topics such as code review requests, logistics, or on other topics not directly related to work.
+See this `community post`_ for why direct messages are considered harmful.
+
+.. _community post: https://community.lsst.org/t/dms-in-slack-considered-harmful/4036
+
+.. _slack-editing:
+
+Editing messages
+----------------
+
+Slack has a feature that allows a user to edit a message sent by that user.
+This feature must be used for  typographical or grammatical errors that the user wishes to correct, or if the message is accidentally sent  before it is fully typed out.
+While Slack is less formal than email, messages should be coherent and in multiple sentences if required, instead of multiple single-line messages in short bursts. This is so that it is easy to link to them elsewhere in the future.
+In place of amending a message, it is preferable to use strikethrough to retract an idea or statement that If you post something you recognize as wrong.
+A corrected statement can be added next to it.
+A message should be deleted only if posted in a wrong channel, or if it is not appropriate in a social forum.
```

Comment:
Why?  If it may be confusing to others who come after, isn't removing it altogether better than striking through it?

Reply:
Is this comment for strikethrough or deleting? I think deleting would be confusing since I might see the message in my notification but not on Slack. Strikethrough keeps a record of what was said, and that it was taken back.